### ISSUE TITLE
[git] return result sorted by date, i.e: newer branches first

### ISSUE THREAD

**Issue by luiskunz** [withfig/autocomplete#757]
### Sanity checks

- [X] My issue relates to a specific CLI completion spec (e.g. `git checkout` is missing options in `git` completion spec). If your issue is more general, please create your issue here: [withfig/fig](https://github.com/withfig/fig/issues/new?assignees=&labels=NEED_TO_LABEL&template=main_issue_template.yml)
- [X] I have searched [github.com/withfig/autocomplete/issues](https://github.com/withfig/autocomplete/issues?q=) and there are no duplicates of my issue
- [ ] I would like to work on this.

### What CLI tool does this relate to?

git

### Which statement makes the most sense?

This completion spec does not exist

### Issue Details

Hello there

First of all, thank you so much for this tool which is a change changer. I am enjoying it a lot so far and am amazed every time I use a new function.

The git auto-complete when I checkout into an existing branch it returns me the list with all of them, the thing is that mostly I want to checkout the newer branch due that mostly is the normal workflow. I did try to dive into the page and google but I couldn't find anything related.

Looking forward to hearing about it!

Best
Luis

**Comment by zp4rker** [comment]
Is there a git command that lists branches sorted by most recently updated? If not, the only way I could see this being possible is manually checking the log of each branch and comparing that however that would be way too janky.

**Comment by luiskunz** [comment]
> ng the log of each branch and comparing that however that would be way to

Not sure about this git command that it should exist. Regarding my issue, it is resolved.

One of my friends at work was showing me this amazing tool and when he did try the branch he have everything unsorted and I decided to come and check around but when I did install and set up everything I did realize that it was working perfectly with sorted branches.

I still love this tool every time I'm in the console, keep doing this wonderful work!

Best,
Luis
